lib/ansible/plugins/inventory/aws_ec2.py

```diff
@@ -134,9 +134,13 @@
     prefix: aws_region
 # Set individual variables with compose
 compose:
-  # Use the private IP address to connect to the host
+  # Use the private IP address to connect to the host and add additional variables as hostvars
+  # EC2 attribute private_ip_address will be mapped to hostvars ansible_host and internal_ip_adddress
```

Comment:
Maybe return these two lines back to the line it was before, and move `# EC2 attribute private_ip_address will be mapped to hostvar internal_ip_adddress` to the line before it is set for clarity.

In the line above `add additional variables as hostvars` seems redundant to line 135. If line 135 isn't clear, I'd modify it to something like `# Set additional variables as hostvars` instead of lumping in a more general comment before setting something specific.